homeassistant/components/ambiclimate/strings.json

```
@@ -7,14 +7,14 @@
       }
     },
     "create_entry": {
-      "default": "Successfully authenticated with Ambiclimate"
+      "default": "[%key:common::config_flow::create_entry::authenticated%]"
     },
     "error": {
       "no_token": "Not authenticated with Ambiclimate",
       "follow_link": "Please follow the link and authenticate before pressing Submit"
     },
     "abort": {
-      "already_configured_account": "[%key:common::config_flow::abort::already_configured_account%]",
+      "already_configured": "[%key:common::config_flow::abort::already_configured_account%]",
       "oauth2_missing_configuration": "[%key:common::config_flow::abort::oauth2_missing_configuration%]",
```

Please rename this key to `missing_configuration`.
```suggestion
      "missing_configuration": "[%key:common::config_flow::abort::oauth2_missing_configuration%]",
```

---

Should this be done to everything?
From
https://github.com/home-assistant/core/blob/6b77633e6e5695436d13fe3d195f417358df51bb/homeassistant/strings.json#L66-L68
To
```json
"missing_configuration": "[%key:common::config_flow::abort::oauth2_missing_configuration%]", 
"authorize_url_timeout": "[%key:common::config_flow::abort::oauth2_authorize_url_timeout%]", 
"no_url_available": "[%key:common::config_flow::abort::oauth2_no_url_available%]", 
```

---

That file are the common strings.